list.html

```diff
@@ -0,0 +1,156 @@
+                <h2>Ordered lists</h2>
+                <p>A responsive image changes its size and adjusts accordingly as the screen size changes</p>
+                <div class="component-container">
+                    <div>
+                        <ol type="1">
+                            <li>Fire</li>
+                            <li>Water</li>
+                            <li>Wind</li>
+                        </ol>
+                    </div>
+                    <div>
+                        <ol type="A">
+                            <li>Earth</li>
+                            <li>Mars</li>
+                            <li>Jupiter</li>
+                        </ol>
+                    </div>
+                    <div>
+                        <ol type="a">
+                            <li>Uchiha</li>
+                            <li>Uzumaki</li>
+                            <li>Senju</li>
+                        </ol>
+                    </div>
+                    <div>
+                        <ol type="I">
+                            <li>Pop</li>
+                            <li>Rock</li>
+                            <li>Alternate</li>
+                        </ol>
+                    </div>
+                    <div>
+                        <ol type="i">
+                            <li>Meta</li>
+                            <li>Google</li>
+                            <li>Microsoft</li>
+                        </ol>
+                    </div>
+
+                </div>
+
+                <h2>Unordered lists</h2>
+                <p>A responsive image changes its size and adjusts accordingly as the screen size changes</p>
+                <div class="component-container">
+                    <div>
+                        <ul class="ul-disc">
+                            <li>Fire</li>
+                            <li>Water</li>
+                            <li>Wind</li>
+                        </ul>
+                    </div>
+                    
+                    
+                    <div>
+                        <ul class="ul-circle">
+                            <li>Pop</li>
+                            <li>Rock</li>
+                            <li>Alternate</li>
+                        </ul>
+                    </div>
+                    <div>
+                        <ul class="ul-square">
+                            <li>Meta</li>
+                            <li>Google</li>
+                            <li>Microsoft</li>
+                        </ul>
+                    </div>
+                
+                </div>
+
+                <h2>Stacked lists</h2>
+                <p>A responsive image changes its size and adjusts accordingly as the screen size changes</p>
+                <div class="component-container">
+                    <div>
+                        <ul>
+                            <li class="list-stacked">
+                                    <div class="avatar-x-small">
```

Comment:
just a suggestion, you can use class name like avatar-xs instead of writing complete word

Reply:
Sure thing, will keep that in mind going forward